content/06.quality_criteria.md

```diff
@@ -250,74 +322,14 @@ especially effective at the source code level.
-    criteria is fulfilled.
-
-* **[QC.Wor02]** New changes in the source code **MUST** be placed in individual branches.
-
-  * **[QC.Wor02.1]** It is **RECOMMENDED** to agree on a branch nomenclature, usually by prefixing,
-    to differentiate change types (e.g. feature, release, fix).
-
-* **[QC.Wor03]** The existence of a secondary long-term branch that contains the changes for the
-  next release is **RECOMMENDED**.
-
-  * **[QC.Wor03.1]** Next release changes **SHOULD** come from the individual branches.
-
-  * **[QC.Wor03.2]** Once ready for release, changes in the secondary long-term branch are merged
-    into the main branch and versioned. At that point in time, main and secondary branches are
-    aligned. This step **SHOULD** mark a production release.
-
-## 4.11. Semantic Versioning [QC.Ver]
+### 4.4.1. Semantic Versioning [QC.Ver]
 
 * **[QC.Ver01]** Semantic Versioning [@https://semver.org] specification is **RECOMMENDED** for
   tagging the production releases.
 
-## 4.12. Code Management [QC.Man]
-
-* **[QC.Man01]** An issue tracking system facilitates structured software development. Leveraging
-  issues to track down both new enhancements and defects (bugs, documentation typos) is
-  **RECOMMENDED**.
-
-  * **[QC.Man01.1]** In addition to monitoring the internal development, issues are the best
-    means for supporting users. External users **SHOULD** be able to create issues based on the
-    operational performance of the software.
-
-  * **[QC.Man01.2]** The description of an issue **SHOULD** be concise and state clearly the
-   problem. It is **RECOMMENDED** to add any reference to the actual problem. In the case of bugs,
-   the issue **SHOULD** be accompanied by the relevant debug information.
-         * The usage of templates for the issue description is **RECOMMENDED**.
-
-* **[QC.Man02]** In social coding environments, pull or merge requests represent the cornerstone of
-  collaboration. A pull or merge request provides a place for review and discussion of the changes
-  proposed to be part of an existing version of the code.
-
-  * **[QC.Man02.1]** Pull/Merge requests **SHOULD** be used for every change in the codebase.
-
-  * **[QC.Man02.2]** A software project **SHOULD** be open to external collaboration through
-    pull/merge requests.
-
-  * **[QC.Man02.3]** A pull/merge request description **SHOULD** be concise and state clearly its
-    purpose (e.g. if it is fixing an observed bug or adding a new feature).
-
-  * **[QC.Man02.4]** The usage of templates for the pull/merge request's description is
-    **RECOMMENDED**.
-
-  * **[QC.Man02.5]** It is **RECOMMENDED** to use pull/merge requests to address open issues.
-
-  * **[QC.Man02.6]** The pull/merge request description **SHOULD** make reference to the identifiers
-    of the issues it is fixing (to eventually close them, either manually or automatically).
-
-## 4.13. Code Review [QC.Rev]
+### 4.4.2. Code Review [QC.Rev]
```

Comment:
"Code Review" under "Code Management"? It is indeed part of the workflow, e.g. create a change >> review it >> approve/reject it

Reply:
agree, moved to 4.1.4. Code Review after 4.1.3. Code Management